package.json

```diff
@@ -66,7 +66,7 @@
     "doc:uml:factory": "sol2uml ./contracts/OtokenFactory.sol -f png -o ./docs/uml/GammaFactory.png",
     "doc:uml:whitelist": "sol2uml ./contracts/Whitelist.sol -f png -o ./docs/uml/GammaWhitelist.png",
     "doc:uml:addressbook": "sol2uml ./contracts/AddressBook.sol -f png -o ./docs/uml/GammaAddressbook.png",
-    "docify": "node docify.js",
+    "docify": "node scripts/docify.js",
```

Comment:
I think we should find another folder for this script, as this folder meant for the scripts that help interacting with the smart contracts IMO.

Reply:
maybe update the folder structure in `scripts/`?

Reply:
Sounds good :+1:

Reply:
Done ✅